reactor-core/src/main/java/reactor/util/context/ContextN.java

```diff
@@ -165,6 +165,16 @@ public void unsafePutAllInto(ContextN other) {
 		other.putAll((Map<Object, Object>) this);
 	}
 
+	/**
+	 * Creates a new {@link ContextN} with values from the provided {@link Map}
```

Comment:
it is actually not creating a _new_ `ContextN`. This javadoc should probably state that even though public, this method is not exposed to end users and can be used to internally modify the state of the `ContextN`.

Reply:
:facepalm: You're absolutely right. I used a draft meant for changes considered for the constructor. Just fixed now.